lib/puppet/provider/cisco_portchannel_global/nxapi.rb

```diff
@@ -0,0 +1,256 @@
+# The NXAPI provider for cisco_portchannel_global
+#
+# December, 2015
+#
+# Copyright (c) 2014-2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'cisco_node_utils' if Puppet.features.cisco_node_utils?
+begin
+  require 'puppet_x/cisco/autogen'
+rescue LoadError # seen on master, not on agent
+  # See longstanding Puppet issues #4248, #7316, #14073, #14149, etc. Ugh.
+  require File.expand_path(File.join(File.dirname(__FILE__), '..', '..', '..',
+                                     'puppet_x', 'cisco', 'autogen.rb'))
+end
+
+Puppet::Type.type(:cisco_portchannel_global).provide(:nxapi) do
+  desc 'The new NXAPI provider.'
+
+  confine feature: :cisco_node_utils
+  defaultfor operatingsystem: :nexus
+
+  mk_resource_methods
+
+  PC_GLOBAL_NON_BOOL_PROPS = [
+    :bundle_hash,
+    :bundle_select,
+    :hash_distribution,
+    :hash_poly,
+    :load_defer,
+    :rotate,
+  ]
+  PC_GLOBAL_BOOL_PROPS = [
+    :asymmetric,
+    :concatenation,
+    :resilient,
+    :symmetry,
+  ]
+  PC_GLOBAL_ALL_PROPS = PC_GLOBAL_NON_BOOL_PROPS + PC_GLOBAL_BOOL_PROPS
+
+  PuppetX::Cisco::AutoGen.mk_puppet_methods(:non_bool, self, '@pc_global',
```

Comment:
Please make the the `@nu`, `nu_obj` changes here like you did for `cisco_interface_portchannel`